### ISSUE TITLE
Encode in PDF417

### ISSUE THREAD

**Issue by hervethouzard** [phonegap/phonegap-plugin-barcodescanner#201]
Hello,

Actually the plugin "only" generate QrCodes, is there a way to modify it so that it encodes in PDF417?

bye

**Comment by macdonst** [comment]
@hervethouzard no, not currently the library we use ZXing can only do QR Codes.
